src/app/src/components/Layers/MunicipalBoundariesLayer.js

```diff
@@ -1,68 +1,37 @@
-import { useEffect, useMemo, useState } from 'react';
 import L from 'leaflet';
+import 'leaflet.vectorgrid';
 
-import { useLayerVisibility, useMapLayer } from '../../hooks';
-import { useSelector } from 'react-redux';
-import { MUNICIPAL_BOUNDARY_LABELS_MIN_ZOOM_LEVEL } from '../../constants';
-import { PANES } from '../../constants';
+import { useToken } from '@chakra-ui/react';
 
-const MUNICIPAL_BOUNDARIES_LAYER_STYLE = {
-    color: '#553C9A', // var(--chakra-colors-purple-700)
-    weight: 1,
-    fillOpacity: '0.1',
-    dashArray: '4',
-};
+import { useLayerVisibility, useMapLayer } from '../../hooks';
+import { MUNICIPAL_BOUNDARIES, PANES } from '../../constants';
 
 export default function MunicipalBoundariesLayer() {
     const showLayer = useLayerVisibility('MUNICIPAL_BOUNDARIES');
 
-    const [layerData, setLayerData] = useState({
-        type: 'FeatureCollection',
-        features: [],
-    });
-
-    useEffect(() => {
-        fetch(`${process.env.PUBLIC_URL}/data/muni.geo.json`)
-            .then(response => response.json())
-            .then(setLayerData);
-    }, []);
-
-    return showLayer && layerData ? <RenderGeoJson json={layerData} /> : null;
+    return showLayer ? <VectorGrid {...MUNICIPAL_BOUNDARIES} /> : null;
 }
 
-function RenderGeoJson({ json }) {
-    const { basemapType, mapZoom } = useSelector(state => state.map);
-
-    // Separating this condition to its own memoized value prevents
-    // re-creation of the layer on every zoom change
-    const shouldShowLabels = useMemo(
-        () => mapZoom > MUNICIPAL_BOUNDARY_LABELS_MIN_ZOOM_LEVEL,
-        [mapZoom]
-    );
-
-    const layer = useMemo(
-        () =>
-            L.geoJSON(json, {
-                style: () => MUNICIPAL_BOUNDARIES_LAYER_STYLE,
-                pane: PANES.MUNICIPAL_BOUNDARIES.label,
-                onEachFeature: shouldShowLabels
-                    ? (feature, layer) => {
-                          layer.bindTooltip(feature.properties.name20, {
-                              permanent: true,
-                              direction: 'center',
-                              className: `muni-label${
-                                  basemapType === 'satellite'
-                                      ? ' muni-label-light'
-                                      : ''
-                              }`,
-                              pane: PANES.MUNICIPAL_BOUNDARY_LABELS.label,
-                          });
-                      }
-                    : null,
-                interactive: false,
+function VectorGrid({ url, maxNativeZoom = 14 }) {
```

Comment:
Elsewhere the `maxNativeZoom` is 12 and that's the max zoom defined when running `tippecanoe` command, is 14 the default because that's more standard for tiles?

Reply:
Good catch! I saw `maxNativeZoom: 14` in the example here: https://leaflet.github.io/Leaflet.VectorGrid/vectorgrid-api-docs.html#vectorgrid-protobuf and mentally internalized it as the default value, which it isn't: it's just a demo value.

Going to switch to `12` here for consistency.

The value of `12` was chosen after some trial and error to keep the tileset size low while also having enough detail for when users are drawing their shapes. After `12` I saw diminishing returns on increased size. At `12` the tileset is about less than 400MB, and costs about $0.05 / month to host with 10K requests, which seemed acceptably affordable.

I'll add that bit to the Ingest README.

Reply:
Fixed in a254405 and ce02f5e